genie-core/src/test/java/com/netflix/genie/core/configs/ServicesConfigTest.java

```diff
@@ -408,4 +417,26 @@ public FactoryBean fileTransferFactory() {
     public JobsProperties jobsProperties() {
         return new JobsProperties();
     }
+
+    /**
+     * Returns a temporary directory as the jobs resource.
+     *
+     * @return The job dir as a resource.
+     * @throws IOException If there is a problem.
+     */
+    @Bean
+    public Resource jobsDir() throws IOException {
+        final File jobsDir = Files.createTempDir();
```

Comment:
Adding this as a `@Rule TemporaryFolder` in `GenieCoreTestApplication` is not going to work because the latter is not a JUnit class and so JUnit annotations such as `@Rule` are simply not detected and processed (i.e. like adding a `@Before` annotation to a method in `JobMonitor.java`, it's just having no effect).